### PR TITLE
Added simple tabs

### PR DESCRIPTION
**Pull request by hannahhanle**

### CODE REVIEW

**Comment by ababaian** [comment]
This is failing to build?

File: palmid.Rmd [hunk 1]
```diff
@@ -310,11 +318,13 @@ plotly::ggplotly(date.report)
...
-Organism meta-data from the SRA sequencing libraries matching the input-virus.
-Word size and color is scaled by proximity of the input-virus to its match in 
-the library. Wordclouds show a) Researcher provided annotations of the libraries
-or b) STAT automated taxonomic (orders) analysis of the reads contained within
-the library ([Katz et al., 2021](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02490-0)).
+Organism meta-data from the SRA sequencing libraries matching the
+input-virus. Word size and color is scaled by proximity of the
+input-virus to its match in the library. Wordclouds show a) Researcher
+provided annotations of the libraries or b) STAT automated taxonomic
+(orders) analysis of the reads contained within the library ([Katz et
```
Comment (ababaian) [inline palmid.Rmd:325]: Weird line break here? Did you run this through a linter?

**Comment by hannahhanle** [comment]
> Can you post the `.html` file for the tabbed output to see what it looks like in comparison?

GitHub doesn't let me upload an HTML directly. Here's the Google Drive [link](https://drive.google.com/file/d/1MY-QKmuDOJKlW2oaEUE1QPxPSEhtCfgJ/view?usp=sharing)!

File: palmid.Rmd (hunk 1 above)
Comment (hannahhanle) [reply palmid.Rmd]: I didn't run through a linter actually. Just pushed fixes to weird line breaks.

**Pull request merged**